SCClassLibrary/Common/Core/Kernel.sc

```diff
@@ -613,7 +613,7 @@ Method : FunctionDef {
 	var <ownerClass, <name, <primitiveName;
 	var <filenameSymbol, <charPos;
 
-	defaultArguments { ^this.prototypeFrame.drop(1).keep(argNames.size - 1).copy }
+	defaultArgs { ^this.prototypeFrame.drop(1).keep(argNames.size - 1).copy }
```

Comment:
no need for an extra copy, because `drop(1)` is guaranteed to return a different array.